packages/utils/parcel-watcher-watchman-js/.gitignore

```diff
@@ -0,0 +1 @@
+/lib
```

Comment:
Lib should already be ignored, and if not it should be fixed in the root `.gitignore` file